docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-running-the-examples-proc.adoc

```diff
@@ -0,0 +1,51 @@
+[id='optimizer-running-the-examples-proc']
+= Running the Business Optimizer examples
+
+.Prerequisite
+Download and install {PRODUCT} from the https://access.redhat.com[Red Hat Customer Portal]. 
+
+.Procedure
+. Download the {PLANNER} file from the https://access.redhat.com[Red Hat Customer Portal].
+. Unzip the {PLANNER} files.
+. Open the directory `examples` and run the following script:
++
+Linux or Mac:
++
+
+[source]
+----
+$ cd examples
+$ ./runExamples.sh
+----
++
+Windows:
++
+
+[source]
+----
+$ cd examples
+$ runExamples.bat
+----
+
+.Download {PLANNER}
+image::PlannerIntroduction/DownloadAndRunTheExamples/distributionZipProductized.png[]
```

Comment:
The image should be removed; it shows some jboss and brms stuff - which is completely obsolete.

Reply:
Removed.